doc/book/app_server/luajit_getmetrics.rst

```diff
@@ -0,0 +1,441 @@
+
+LuaJIT getmetrics
+=================
+
+Tarantool can return metrics of a current instance via the Lua API or the C API.
+
+..  contents::
+    :local:
+    :depth: 2
+
+.. _luajit_getmetrics_getmetrics:
+
+misc.getmetrics()
+~~~~~~~~~~~~~~~~~
+
+.. function:: getmetrics()
+
+    Get the metrics values into a table.
+   
+    Parameters: none
+     
+    :return: table
+
+    Example: ``metrics_table = misc.getmetrics()``  
+
+.. _luajit_getmetrics_tablevalues:
+
+getmetrics table values
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The metrics table contains 19 values.
+All values have type = 'number' and are the result of a cast to double, so there may be a very slight precision loss.
+Values whose names begin with ``gc_`` are associated with the
+`LuaJIT garbage collector <http://wiki.luajit.org/New-Garbage-Collector/>`_
```

Comment:
**** If Buristan does not comment, I will not make a change.

Reply:
Re the general comment "One general comment: is it intentional, that you don't leave the link to this page in the metrics doc, as we discussed in #1597?" ...
Later I added this comment in #1597:
Re the earlier discussion (above) about the feature that is so similar
https://www.tarantool.io/en/doc/latest/book/monitoring/metrics_reference/#luajit-metrics
Originally I wrote:
Note: Although value names are similar to value names in LuaJIT metrics, and the values are exactly the same, misc.getmetrics() is slightly easier because there is no need to ‘require’ the misc module."
... But then I threw it away because the .rst file related [to] those LuaJIT metrics has disappeared from the documentation sources.

Reply:
Sorry for the long answer.

I suppose that the best way is to mention the following link to [the Lua Wiki about Garbage Collector](http://lua-users.org/wiki/EmergencyGarbageCollector) (see the Detailed description section). It describes all steps except the atomic step. It is slightly described by Roberto on [this workshop](https://www.lua.org/wshop18/Ierusalimschy.pdf#page=16). We can mention the second link here too.

Reply:
Re the link to the page in the metrics doc: I was wrong, the page was not really missing, I had failed to copy submodule files. So I have added the "Note:" about it that we discussed in #1597.
Re the suggestion to mention the links to the Lua wiki and the slide: okay, I did that.